subsys/bluetooth/host/hci_core.c

```diff
@@ -140,7 +140,7 @@ struct cmd_data {
 static struct cmd_data cmd_data[BT_BUF_CMD_TX_COUNT];
 
 #define cmd(buf) (&cmd_data[net_buf_id(buf)])
-#define acl(buf) ((struct acl_data *)net_buf_user_data(buf))
+#define acl(buf) ((struct bt_conn_rx *)net_buf_user_data(buf))
```

Comment:
To be explicit that this is the user data in the net buf...

And this is only used for Controller to  Host data Flow Control, right? should implementation around this be conditionally compile-able?

```suggestion
#define acl_rx_user_data(buf) ((struct bt_conn_rx *)net_buf_user_data(buf))
```

Reply:
ok... did not look in full detail... but if it is already guarded by conditional compile, then ignore it. But do consider a naming to reflect that it is net buf user (/meta) data.